src/components/v2/V2Project/V2BalancesModal/V2BalancesModal.tsx

```diff
@@ -0,0 +1,171 @@
+import { SettingOutlined } from '@ant-design/icons'
+import { BigNumber } from '@ethersproject/bignumber'
+import { Button, Modal, Space } from 'antd'
+import { ProjectMetadataV4 } from 'models/project-metadata'
+import { TokenRef } from 'models/token-ref'
+import { useEffect, useState } from 'react'
+import { uploadProjectMetadata } from 'utils/ipfs'
+import { t, Trans } from '@lingui/macro'
+import { TransactorInstance } from 'hooks/Transactor'
+import { V2ProjectTokenBalance } from 'components/v2/shared/V2ProjectTokenBalance'
+import ERC20TokenBalance from 'components/v1/shared/ERC20TokenBalance'
+
+import { useForm } from 'antd/lib/form/Form'
+
+import { V2_PROJECT_IDS } from 'constants/v2/projectIds'
+import V2TokenRefs from './V2TokenRefs'
+import { AssetInputType } from './V2TokenRefs'
+
+export interface EditTrackedAssetsForm {
+  tokenRefs: { assetInput: { input: string; type: AssetInputType } }[]
+}
+
+export function V2BalancesModal({
```

Comment:
UX feedback

- middle-align the (x)
- change it to a "trash" icon or the (-) icon
- Add a tooltip to the trash icon button saying something like "Delete token" or "Untrack token"
- (stretch) use the `Button` element for accessibility (try the "link" button type for styling)

<img width="562" alt="Screen Shot 2022-06-23 at 7 39 56 AM" src="https://user-images.githubusercontent.com/94939382/175161063-919a27b2-4461-4942-aedd-d3c1776cbf99.png">

Reply:
I like trash I think so lets give that a go!

Reply:
<img width="656" alt="image" src="https://user-images.githubusercontent.com/104132113/175758048-47e52f66-e847-49a3-a078-550d0c6a7a9d.png">

Reply:
am using style=text button isntead as the button is same ui color until hovered (as above)

<img width="622" alt="image" src="https://user-images.githubusercontent.com/104132113/175758065-992b771f-66f7-44df-9980-f17a5dbb4392.png">

Reply:
<img width="604" alt="image" src="https://user-images.githubusercontent.com/104132113/175758102-3c8d4b00-7359-4663-ac43-8a914626997b.png">
IMO minus looks wrong. For me, minus is like collapsing a menu

Reply:
I was thinking maybe this one

<img width="165" alt="Screen Shot 2022-06-25 at 4 00 55 PM" src="https://user-images.githubusercontent.com/94939382/175761506-a94cc5e3-3a0b-47f2-8777-59cbede1c57d.png">
 
but trash probabl fine